domenetjenester/person/src/main/java/no/nav/foreldrepenger/domene/person/pdl/PersonBasisTjeneste.java

```diff
@@ -64,13 +58,13 @@ public PersoninfoVisning hentVisningsPersoninfo(FagsakYtelseType ytelseType, Akt
         var query = new HentPersonQueryRequest();
         query.setIdent(aktørId.getId());
         var projection = new PersonResponseProjection()
-            .falskIdentitet(new FalskIdentitetResponseProjection().erFalsk())
+            .folkeregisteridentifikator(new FolkeregisteridentifikatorResponseProjection().identifikasjonsnummer())
             .navn(new NavnResponseProjection().fornavn().mellomnavn().etternavn())
             .adressebeskyttelse(new AdressebeskyttelseResponseProjection().gradering());
 
         var person = pdlKlient.hentPerson(ytelseType, query, projection);
 
-        sjekkFalskIdentitet(person, ytelseType, aktørId);
```

Comment:
Hva er bakgrunnen for at du endrer fra å gjøre ett kall to til kall mot pdl?

Reply:
PDL rasker sammen et svar basert på det vi spør om. 
Gir ikke mening å alltid spørre om falskidentitet. Mens identifikator noe som er mer naturlig å bruke.
Manglende identifikator tilsvarer manglende navn/fødselsdato men vi kjenner ikke korrelasjon.